setup.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio JSONSchemas.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Invenio JSONSchemas"""
+
+import os
+import sys
+
+from setuptools import setup
+from setuptools.command.test import test as TestCommand
+
+readme = open('README.rst').read()
+history = open('CHANGES.rst').read()
+
+requirements = [
+    'six>=1.7.2',
+    'Invenio>=2.1.0',
+    'jsonpointer>=1.9',
+    'jsonschema>=2.5.0',
+]
+
+test_requirements = [
+    'pytest>=2.7.0',
+    'pytest-cov>=1.8.0',
+    'pytest-pep8>=1.0.6',
+    'coverage>=3.7.1',
+]
+
+
+class PyTest(TestCommand):
+    """PyTest Test."""
+
+    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
+
+    def initialize_options(self):
+        """Init pytest."""
+        TestCommand.initialize_options(self)
+        self.pytest_args = []
+        try:
+            from ConfigParser import ConfigParser
+        except ImportError:
+            from configparser import ConfigParser
+        config = ConfigParser()
+        config.read('pytest.ini')
+        self.pytest_args = config.get('pytest', 'addopts').split(' ')
+
+    def finalize_options(self):
+        """Finalize pytest."""
+        TestCommand.finalize_options(self)
+        self.test_args = []
+        self.test_suite = True
+
+    def run_tests(self):
+        """Run tests."""
+        # import here, cause outside the eggs aren't loaded
+        import pytest
+        import _pytest.config
+        pm = _pytest.config.get_plugin_manager()
+        pm.consider_setuptools_entrypoints()
+        errno = pytest.main(self.pytest_args)
+        sys.exit(errno)
+
+# Get the version string. Cannot be done with import!
+g = {}
+with open(os.path.join('invenio_jsonschemas', 'version.py'), 'rt') as fp:
+    exec(fp.read(), g)
+    version = g['__version__']
+
+setup(
+    name='Invenio JSONSchemas',
```

Comment:
Yes, setup's `name` should be `invenio-jsonchemas`, again following `invenio-query-parser` prior art example, as I mentioned above.  (Also, the PR uses "Invenio JSONSchemas" while it should rather be "Invenio-JSONSchemas", etc.)